improver_tests/calibration/dataframe_utilities/test_dataframe_utilities.py

```diff
@@ -165,19 +165,36 @@ def setUp(self):
             np.array(self.height, dtype=np.float32), "height", units="m",
         )
 
-        self.forecast_df_station_id = self.forecast_df.copy()
-        self.forecast_df_station_id = self.forecast_df_station_id.loc[
-            self.forecast_df_station_id["wmo_id"].isin(self.wmo_ids[:-1])
+        df = self.forecast_df
+        condition = (df["time"].isin([self.time1, self.time2])) & (
+            df["wmo_id"] == self.wmo_ids[2]
+        )
+        self.forecast_df_multi_station_id = df.drop(df[condition].index)
+        self.forecast_df_multi_station_id["station_id"] = (
+            self.forecast_df_multi_station_id["wmo_id"] + "0"
+        )
+        df = self.truth_subset_df
+        condition = (df["time"].isin([self.time1, self.time2])) & (
+            df["wmo_id"] == self.wmo_ids[2]
+        )
+        self.truth_df_multi_station_id = df.drop(df[condition].index)
+        self.truth_df_multi_station_id["station_id"] = (
+            self.truth_df_multi_station_id["wmo_id"] + "0"
+        )
+
+        self.forecast_df_one_station_id = self.forecast_df.copy()
+        self.forecast_df_one_station_id = self.forecast_df_one_station_id.loc[
+            self.forecast_df_one_station_id["wmo_id"].isin(self.wmo_ids[:-1])
```

Comment:
These "one_station_id" dataframes actually have two wmo_ids in them. Is that what you want?

Reply:
I've added a comment to clarify.